src/docfx/lib/markdown/MarkdownEngine.cs

```diff
@@ -170,6 +170,27 @@ public string ToPlainText(MarkdownObject markdownObject)
             return sb.ToString();
         }
 
+        private string? GetLinkText(string url, MarkdownObject markdownObject)
+        {
+            if (markdownObject is LinkInline {IsImage: false} link)
+            {
+                    if (link.FirstChild != null && link.FirstChild.NextSibling == null)
+                    {
+                        return link.FirstChild.ToString();
+                    }
+                    if (link.FirstChild == null)
+                    {
+                        return link.Title;
+                    }
+            }
+            else if (markdownObject is HtmlInline aLink)
+            {
+                return ToPlainText(aLink.Parent);
```

Comment:
Will this method return the link text reliably?

Reply:
I will remove it.